Mathlib/Tactic/StacksAttribute.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Attributes
+
+/-!
+The `stacks` attribute.
+-/
+
+open Lean
+
+namespace Mathlib.Stacks
+
+/--
+The syntax for a Stacks tag: it is an optional number followed by an optional identifier.
+This allows `044Q3` and `GH3F6` as possibilities.
```

Comment:
Maybe worth linking to the Stacks project page on tags: https://stacks.math.columbia.edu/tags

Reply:
Added, thanks!